core/src/avm2/verify.rs

```diff
@@ -302,6 +330,12 @@ pub fn verify_method<'gc>(
                             1078,
                         )?));
                     }
+
+                    assert!(bytes_read > 1);
+                    byte_info[previous_position as usize] =
+                        ByteInfo::OpStart(AbcOp::FindPropStrict { index });
+                    byte_info[(previous_position + 1) as usize] =
```

Comment:
can you document that it's a slightly hacky way to emit multiple ops, with explanation how it works?